src/config/dark-sites.config

```diff
@@ -772,6 +770,7 @@ totalwarwarhammer.gamepedia.com
 tracker.fumik0.com
 tracr.co
 training.azeria-labs.com
+translate.metalune.xyz
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/124384433-7ee2cb00-dcd1-11eb-86c6-95e4823e7d9c.png)
Not dark by default.